src/vtk6-1-fixes.patch

```diff
@@ -0,0 +1,86 @@
+This file is part of MXE.
+See index.html for further information.
+
+Contains ad hoc patches for cross building.
+
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Martin=20M=C3=BCllenhaupt?= <muellni@netlair.de>
```

Comment:
Git escaped Unicode characters in the author name (Martin Müllenhaupt <muellni@netlair.de>).